recipes/apptek-callcenter-dialogues/reproduce_parakeet_v3.py

```
@@ -0,0 +1,474 @@
+
+    import torch
+
+    asr_model = _load_asr_model(args)
+
+    with torch.inference_mode():
+        for offset in tqdm(range(0, len(missing), args.batch_size), desc="Parakeet direct transcription"):
+            batch = missing[offset : offset + args.batch_size]
+            audio_paths = [str(_manifest_audio_path(row)) for row in batch]
+            texts = _transcribe_batch(asr_model, audio_paths, args)
+            prediction_rows = []
+            for source_row, text in zip(batch, texts):
```

Ruff (0.15.14)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)